how-to-respond-nl.md

```diff
@@ -0,0 +1,43 @@
+| <a href="README.md">Home</a> | <a href="introduction-nl.md">Inleiding</a> | <a href="questions-nl.md">Onze voorstellen en vragen</a> | Reageren op de marktconsultatie  |
+| -------- | -------- | -------- | -------- |
+
+# Hoe te reageren
+
+Op deze pagina wordt het proces en het tijdschema voor de reactie op de marktconsultatie toegelicht.
+
+## Uitgangspunten marktconsultatie
+
+Met OpenZaak realiseren we een belangrijke stap om de [Common Ground](https://commonground.nl/cms/view/54476259/over-common-ground) principes te realiseren. Maar wij realiseren ons goed dat het pas van waarde is als het actief wordt gebruikt en doorontwikkelt.
+
+Onze motivatie is een community te realiseren waarin overheid, leveranciers en partners samenwerken. Een community, zonder lock-ins, die voor alle betrokken werkt. Vanuit die gezamenlijkheid willen we ook deze marktconsultatie opzetten.
+
+We willen jullie ervaring, inspiratie en denkkracht inzetten om tot de juiste opzet te komen. Om het goede gesprek te kunnen voeren hebben we in dit document onze behoeften, voorstellen en vragen aan de markt uitgewerkt.
+
+## Procedure van de marktconsultatie
+
+De marktconsultatie bestaat uit twee onderdelen.
+
+1. Een schriftelijke reactie op onze vragen. We stellen bewust geen restricties aan de vorm en lengte van jullie antwoorden. Deze reacties dienen uiterlijk op 1-5-2020 vóór 17:00 uur ontvangen te zijn. U kunt uw antwoorden sturen aan niels.lindeboom@dimpact.nl. Op basis van jullie antwoorden gaan wij opnieuw kijken naar ons voorstel en dit aanpassen als er goede suggesties zijn gegeven. Dit aangepaste voorstel wordt dan het uitgangspunt voor de workshop (deel twee van de consultatie).
```

Comment:
Dit gebeurt in alle Nederlandse versies - heb jij een voorkeur?

Reply:
voorkeur voor u/uw

Reply:
@hugo-ter-doest, we're going to go ahead and merge #5 without making this change right now - the typo fixes are urgent (because [typos lead to decreased trust](https://www.bbc.co.uk/news/education-14130854)) while jullie/uw is only stylistic. We'll pick this up later this week.

Reply:
I've logged this in #8.